README.md

```diff
@@ -122,6 +122,7 @@ CLAs and best practices for working with GitHub.
 The following users have deployed JanusGraph in production.
 
 * [CELUM](https://www.celum.com/) - [use case and system architecture](https://www.celum.com/en/graph-driven-and-reactive-architecture)
+* [Crédit Agricole CIB](https://www.ca-cib.com/)
```

Comment:
Could you please also add a link to your use-case?
I.e. something like this:

[Crédit Agricole CIB](https://www.ca-cib.com/) - [use-case](https://github.com/JanusGraph/janusgraph/discussions/2734)

Reply:
done